catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx

```diff
@@ -86,13 +87,16 @@ const Overlay = styled.div.attrs({})`
   background: rgba(0, 0, 0, 0.5);
 `;
 
-const Modal = styled(Space).attrs({
+const Modal = styled(Space).attrs(props => ({
   v: { size: 'xl', properties: ['padding-top', 'padding-bottom'] },
   h: { size: 'xl', properties: ['padding-left', 'padding-right'] },
+  role: 'dialog',
+  'aria-labelledby': props.ariaLabelledBy,
```

Comment:
I think this should be applied as it's applied on the JSX element. We could probably move the role there too.